src/net/events.ts

```diff
@@ -560,3 +561,48 @@ export function addEventComponents(em: EntityManager) {
   em.addSingletonComponent(DetectedEventsDef);
   em.addSingletonComponent(EventsDef);
 }
+
+export function eventWizard<ES extends EDef<any>[], Extra>(
```

Comment:
Any reason not to make the 🧙 the only way to register an event handler? We don't actually lose any expressivity, do we?

Reply:
We don't, just didn't want to switch everything unless u think its good too